.github/workflows/_ci.yaml

```
@@ -0,0 +1,203 @@
+name: CI
+run-name: CI-${{ inputs.ARCHITECTURE }}
+
+on:
+  workflow_call:
+    inputs:
+      ARCHITECTURE:
+        type: string
+        required: true
+      CUDA_IMAGE:
+        type: string
+        required: true
+      SRC_JAX:
+        type: string
+        required: true
+      SRC_XLA:
+        type: string
+        required: true
+      SRC_TE:
+        type: string
+        required: true
+      SRC_T5X:
+        type: string
+        required: true
+      SRC_PAXML:
+        type: string
+        required: true
+      SRC_PRAXIS:
+        type: string
+        required: true
+    outputs:
+      TAG_BASE:
+        description: "Tags of the base image built"
+        value: ${{ jobs.build-base.outputs.DOCKER_TAGS }}
```

I think this should be:
```suggestion
        value: ${{ jobs.build-base.outputs.DOCKER_TAG }}
```

---

Fixed in 6501ae905fad29b4869eba7c7a1f749b088cf616